index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="./css/mainstyles.css">
+    <link rel="stylesheet" href="./css/tabletstyles.css">
+    <link rel="stylesheet" href="./css/mobilecss.css">
```

Comment:
```suggestion
    <link rel="stylesheet" href="./css/mobile.css">
```

можно просто `mobile.css` :)